tests/testSetup.js

```diff
@@ -0,0 +1,7 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// To make enzyme work we need to configure Adapter. According
+// to the doc this Adapter have additional peer dependencies,
+// in our case these dependencies are react, react-dom and react-test-renderer.
```

Comment:
I don't think there's much sense in mentioning of peer dependencies – they may change in future while this comment will be here. :P

Also, I'd somehow rewrite it to say: 

```javascript
// Enzyme is React version agnostic and can work with any version through
// adapters system. Enzyme Adapter is a sort of engine that knows how to
// manipulate underlying UI library (it can be used not only with React).
// Since we use React 16, we need to set a proper adapter.
```